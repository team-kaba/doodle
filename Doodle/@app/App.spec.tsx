import 'react-native';

import {render, waitFor} from '@testing-library/react-native';
import React from 'react';

import App from '@app/App';

describe('App.tsx', () => {
  it('renders correctly', async () => {
    const app = render(<App />);
    await waitFor(() => app.getByTestId(':screens/home'));
    expect(app.toJSON()).toMatchSnapshot();
  });
});
