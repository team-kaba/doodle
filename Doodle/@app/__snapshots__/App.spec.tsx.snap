// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.tsx renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    onAppear={[Function]}
    onDisappear={[Function]}
    onDismissed={[Function]}
    onWillAppear={[Function]}
    onWillDisappear={[Function]}
    replaceAnimation="pop"
    stackPresentation="push"
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  >
    <View
      backgroundColor="rgb(255, 255, 255)"
      color="rgb(0, 122, 255)"
      hidden={true}
      title="Main"
      titleColor="rgb(28, 28, 30)"
      topInsetEnabled={true}
      translucent={false}
    />
    <View
      style={
        Array [
          Object {
            "flex": 1,
          },
          Object {
            "backgroundColor": "rgb(242, 242, 242)",
          },
          undefined,
        ]
      }
    >
      <View
        style={
          Object {
            "flex": 1,
          }
        }
      >
        <View
          onAppear={[Function]}
          onDisappear={[Function]}
          onDismissed={[Function]}
          onWillAppear={[Function]}
          onWillDisappear={[Function]}
          replaceAnimation="pop"
          stackPresentation="push"
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        >
          <View
            backgroundColor="rgb(255, 255, 255)"
            color="rgb(0, 122, 255)"
            hidden={true}
            title="Doodle"
            titleColor="rgb(28, 28, 30)"
            topInsetEnabled={true}
            translucent={false}
          />
          <View
            style={
              Array [
                Object {
                  "flex": 1,
                },
                Object {
                  "backgroundColor": "rgb(242, 242, 242)",
                },
                undefined,
              ]
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#2089dc",
                  "bottom": 0,
                  "flex": 1,
                  "justifyContent": "center",
                  "left": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                }
              }
              testID=":screens/home"
            >
              <View
                style={
                  Object {
                    "flex": 1,
                    "justifyContent": "center",
                    "opacity": AnimatedValue {
                      " __value": 1,
                    },
                    "transform": Array [
                      Object {
                        "translateX": AnimatedValue {
                          " __value": -318.75,
                        },
                      },
                    ],
                  }
                }
              >
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": undefined,
                        "fontSize": 12,
                      },
                      Object {
                        "color": "#fefefe",
                        "fontSize": 100,
                      },
                      Object {
                        "fontFamily": "FontAwesome5Free-Solid",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {
                        "fontWeight": "700",
                      },
                    ]
                  }
                >
                  
                </Text>
              </View>
              <Text
                style={
                  Object {
                    "color": "#fefefe",
                    "fontSize": 50,
                  }
                }
              >
                Hello, World!
              </Text>
              <View
                style={
                  Object {
                    "marginVertical": 20,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#fefefe",
                      "fontSize": 30,
                    }
                  }
                >
                  Swipe right to show menu
                </Text>
              </View>
              <View
                style={
                  Object {
                    "flex": 1,
                    "justifyContent": "center",
                    "opacity": AnimatedValue {
                      " __value": 1,
                    },
                    "transform": Array [
                      Object {
                        "translateX": AnimatedValue {
                          " __value": -318.75,
                        },
                      },
                    ],
                  }
                }
              >
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": undefined,
                        "fontSize": 12,
                      },
                      Object {
                        "color": "#fefefe",
                        "fontSize": 100,
                      },
                      Object {
                        "fontFamily": "FontAwesome5Free-Solid",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {
                        "fontWeight": "700",
                      },
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
